<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN2070121W2" ignored="0" name="OtherGoalUserSignUpJourneyMoreThan55Test" tests="5" failures="2" timestamp="18 Aug 2018 16:31:48 GMT" time="0.293" errors="0">
  <testcase name="t3" time="0.212" classname="com.ey.test3po.userjourney.test"/>
  <testcase name="tt2" time="0.006" classname="com.ey.test3po.userjourney.test">
    <skipped/>
  </testcase> <!-- tt2 -->
  <testcase name="tt2" time="0.002" classname="com.ey.test3po.userjourney.test">
    <skipped/>
  </testcase> <!-- tt2 -->
  <testcase name="tt2" time="0.002" classname="com.ey.test3po.userjourney.test">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.ey.test3po.userjourney.test.tt2(test.java:12)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tt2 -->
  <testcase name="tt2" time="0.029" classname="com.ey.test3po.userjourney.test">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.ey.test3po.util.TestListener.onTestFailure(TestListener.java:56)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- tt2 -->
</testsuite> <!-- OtherGoalUserSignUpJourneyMoreThan55Test -->
